app/slackimport.go

```
@@ -845,30 +845,45 @@ func (a *App) oldImportUser(team *model.Team, user *model.User) *model.User {
 }
 
 func (a *App) oldImportChannel(channel *model.Channel, sChannel SlackChannel, users map[string]*model.User) *model.Channel {
-	if channel.Type == model.CHANNEL_DIRECT {
-		sc, err := a.createDirectChannel(users[sChannel.Members[0]].Id, users[sChannel.Members[1]].Id)
+	switch {
+	case channel.Type == model.CHANNEL_DIRECT:
+		if len(sChannel.Members) < 2 {
+			return nil
+		}
+		u1 := users[sChannel.Members[0]]
+		u2 := users[sChannel.Members[1]]
+		if u1 == nil || u2 == nil {
+			return nil
```

Should we log an error here instead of silently failing?